features/format.feature

```diff
@@ -446,11 +446,11 @@ Feature: Custom formats
         When we run "jrnl --export yaml -o {cache_dir}"
         Then the cache should contain the files
         """
-        2020-08-29_entry-the-first.md
-        2020-08-31_a-second-entry-in-what-i-hope-to-be-a-long-series.md
-        2020-09-24_the-third-entry-finally-after-weeks-without-writing.md
+        2020-08-29_entry-the-first.yaml
+        2020-08-31_a-second-entry-in-what-i-hope-to-be-a-long-series.yaml
+        2020-09-24_the-third-entry-finally-after-weeks-without-writing.yaml
         """
-        And the content of file "2020-08-29_entry-the-first.md" in the cache should be
+        And the content of file "2020-08-29_entry-the-first.yaml" in the cache should be
```

Comment:
Could you implement your tests in the pytest-bdd instead of behave? We're trying to phase out behave. See `/tests/bdd`

Reply:
I haven't added any tests, only adjusted the (exiting) tests so they pass. Both the behave and pytest-bdd versions.

Moving the test suite to pytest-bdd is beyond the intended scope of this PR.

Reply:
The added tests have been added to the pytest setup.